query/src/main/java/org/infinispan/query/dsl/embedded/impl/BaseIckleFilterIndexingServiceProvider.java

```diff
@@ -258,7 +260,13 @@ public void onFilterResult(Object userContext, Object eventType, Object instance
                   conversionDone = true;
                }
 
-               invocation.invokeNoChecks(new EventWrapper<>(event.getKey(), event), false, filterAndConvert, true);
+               // TODO: We need a way to propagate the CompletionStages down to the caller - so continuous query
+               // can be non blocking This is to be fixed in https://issues.jboss.org/browse/ISPN-9729
```

Comment:
Missing period. The 2nd line in a TODO should start with 2 spaces, that way IntelliJ shows it as a single TODO.

Reply:
didn't know that, thanks.

Reply:
Also a period in a TODO is a bit nitpicky :D

Reply:
Yeah, I don't think I would have said anything if you didn't put the `-` on the first line. But I also wanted to share the wisdom about the latest changes in IntelliJ's TODO handling :D